ci/tests/general/general_alba.py

```diff
@@ -92,36 +106,23 @@ def prepare_alba_backend(name=None):
         :param name: Name for the backend
         :return: None
         """
-        # @TODO: Fix this, because backend_type should not be configurable if you always create an ALBA backend
-        # @TODO 2: Get rid of these asserts, any test (or testsuite) should verify the required params first before starting execution
-        autotest_config = General.get_config()
         if name is None:
-            name = autotest_config.get('backend', 'name')
-        nr_of_disks_to_claim = autotest_config.getint('backend', 'nr_of_disks_to_claim')
-        type_of_disks_to_claim = autotest_config.get('backend', 'type_of_disks_to_claim')
-        assert name, "Please fill out a valid backend name in autotest.cfg file"
+            name = 'marie'
```

Comment:
Why is this now hard-coded? It seems it used to be loaded from a configuration file.